From ba8d9b8321dd40f157753247111b0a969152af34 Mon Sep 17 00:00:00 2001
From: Ilya Smirnov <ismirno@us.ibm.com>
Date: Mon, 16 Jul 2018 16:25:00 -0500
Subject: [PATCH] Mark IPC_DATA_INVALID Errors as Informational

IPC_DATA_INVALID errors show up on single-node systems (ZZ,
Zeppelin) intermittently without a known or obvious cause.
This change lowers the severity of those errors to
Informational, as they are meaningless on one-node systems
and have never been seen on multinode systems.

Change-Id: I1fd40f7482ef947a3f3a9b33b92832a311418fd4
CQ: SW436958
Reviewed-on: http://rchgit01.rchland.ibm.com/gerrit1/62624
Tested-by: Jenkins Server <pfd-jenkins+hostboot@us.ibm.com>
Tested-by: Jenkins OP Build CI <op-jenkins+hostboot@us.ibm.com>
Tested-by: Jenkins OP HW <op-hw-jenkins+hostboot@us.ibm.com>
Tested-by: FSP CI Jenkins <fsp-CI-jenkins+hostboot@us.ibm.com>
Reviewed-by: Daniel M. Crowell <dcrowell@us.ibm.com>
---
 src/usr/intr/intrrp.C | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/usr/intr/intrrp.C b/src/usr/intr/intrrp.C
index 964aaea..4db6786 100644
--- a/src/usr/intr/intrrp.C
+++ b/src/usr/intr/intrrp.C
@@ -1063,7 +1063,7 @@ void IntrRp::msgHandler()
                                    " an invalid state. msg_queue_id = 0x%lx",
                                    KernelIpc::ipc_data_area.msg_queue_id);
                         /*@ errorlog tag
-                         * @errortype       ERRL_SEV_PREDICTIVE
+                         * @errortype       ERRL_SEV_INFORMATIONAL
                          * @moduleid        INTR::MOD_INTRRP_IPC
                          * @reasoncode      INTR::RC_IPC_DATA_INVALID
                          * @userdata1       IPC Data Area MSG Queue ID
@@ -1073,7 +1073,7 @@ void IntrRp::msgHandler()
                          */
                         l_err = new ERRORLOG::ErrlEntry
                             (
-                             ERRORLOG::ERRL_SEV_PREDICTIVE,    // severity
+                             ERRORLOG::ERRL_SEV_INFORMATIONAL,    // severity
                              INTR::MOD_INTRRP_IPC,             // moduleid
                              INTR::RC_IPC_DATA_INVALID,        // reason code
                              KernelIpc::ipc_data_area.msg_queue_id,
-- 
1.9.1

